src/js/components/award/shared/activity/ContractGrantActivity.jsx

```diff
@@ -46,17 +67,160 @@ const ContractGrantActivity = ({
+            },
+            {
+                title: 'Obligated Amount',
+                paragraphs: [
+                    `${formatMoney(transaction.federal_action_obligation)}`
+                ]
+            },
+            {
+                title: 'Total Obligations to Date',
+                paragraphs: [
+                    `${formatMoney(transaction.running_obligation_total_to_date)} 
+                    ${totalObligation ? `(${calculateTreemapPercentage(transaction.running_obligation_total_to_date, totalObligation)} of Potential Award Amount)` : ''}`
+                ]
+            },
+            {
+                title: 'Action Type',
+                paragraphs: [
+                    `${transaction.type_description || '--'}`
+                ]
+            },
+            {
+                title: 'Description',
+                paragraphs: [
+                    `${transaction.description || '--'}`
+                ]
+            }
+        ]
+    })
+    ));
+
+    const transactionTooltipData = (data) => ({
+        styles: { // y position is in the middle of the line
+            position: 'absolute',
+            transform: `translate(${data.cx + 11}px,${data.cy - 13}px)`
+        },
+        tooltipComponent: <PaginatedTooltipContainer
+            data={transactionTooltipInfo(data.data, 'Modification')}
+            tooltipElement={<Tooltip />} />
+    });
+
+    const handleTooltipDataLine = (data, text) => {
+        let tooltipInfo = null;
+        if (!text) { // potential award amount line
+            tooltipInfo = potentialAwardAmountLineTooltipData(data);
+        }
+        else { // all other award lines
+            tooltipInfo = verticalLinesTooltipData(data, text);
+        }
+        setTooltipData(tooltipInfo);
+        setShowTooltipLine(true);
+    };
+
+    const handleTooltipDataTransaction = (data) => {
+        setTooltipData(transactionTooltipData(data));
+        setShowTooltipTransaction(true);
+    };
+
+    const onMouseMoveTooltip = () => {
+        setHoveringOverTransactionTooltip(true);
+    };
+    const onMouseLeaveTooltip = () => {
+        setHoveringOverTransactionTooltip(false);
+    };
+
+    const showTransactionTooltip = (data, text) => handleTooltipDataTransaction(data, text);
+
+    const hideTooltipTransaction = () => setTimeout(() => setShowTooltipTransaction(false), 500);
+
+    const showHideTooltipLine = (data, text) => {
+        // hide tooltip
+        if (!data && showTooltipLine) {
+            setShowTooltipLine(false);
+            setThisLineOrTextIsHovered('');
+        }
+        else {
+            handleTooltipDataLine(data, text);
+        }
+    };
 
     return (
-        <div ref={divReference} className="contract-grant-activity-visualization">
+        <div ref={divReference} className="award-amounts-viz contract-grant-activity-visualization">
+            {(showTooltipLine || showTooltipTransaction || isHoveringOverTransactionTooltip) && <TooltipWrapper
```

Comment:
I understand what you're doing here w/ the tooltip and why based on our previous convoz. Wondering if we can bake this into the tooltip component itself so that it continues to show when you hover over the tooltip component itself. Can you comment on how we can do this?